javascript/packages/orchestrator/src/cmdGenerator.ts

```diff
@@ -270,7 +270,7 @@ export async function genCmd(
     const parachainIdArgIndex = args.findIndex((arg) =>
       arg.includes("--parachain-id"),
     );
-    args.splice(parachainIdArgIndex, 1);
+    if( parachainIdArgIndex >=0) args.splice(parachainIdArgIndex, 1);
```

Comment:
```suggestion
    if( parachainIdArgIndex >=0 ) args.splice(parachainIdArgIndex, 1);
```

Reply:
Is this something that the `linting` should `fix` or at least catch?

Reply:
fixed by linter.